interactions/client/bot.py

```diff
@@ -1321,12 +1321,16 @@ def load(
             self._extensions[_name] = module
             return extension
 
-    def remove(self, name: str, package: Optional[str] = None) -> None:
+    def remove(
+        self, name: str, remove_commands: Optional[bool] = True, package: Optional[str] = None
```

Comment:
```suggestion
        self, name: str, remove_commands: bool = True, package: Optional[str] = None
```
Optional is Union[x, None]